db/schema.rb

```diff
@@ -176,6 +176,13 @@
     t.boolean "partner_dwp_challenge"
     t.integer "applicant_age"
     t.boolean "non_means_tested"
+    t.string "link_type"
```

Comment:
Where did `link_type` string come from? I didn't see it in the above migration.

Reply:
👍 thanks removed